lib/gitsh/argument_list.rb

```diff
@@ -0,0 +1,21 @@
+module Gitsh
+  class ArgumentList
+    def initialize(args)
+      @args = args
+    end
+
+    def length
+      args.length
+    end
```

Comment:
It's used by the command classes to verify that their argument lists contain the correct number of arguments (e.g. [internal_command.rb:65](https://github.com/thoughtbot/gitsh/blob/5136aa62b1c2c6fd233bd97cac760b45de7326af/lib/gitsh/commands/internal_command.rb#L65)).